docs/codeql/codeql-language-guides/codeql-library-for-actions.rst

```diff
@@ -116,16 +116,17 @@ See the GitHub Actions documentation on `workflows <https://docs.github.com/en/a
    * - CodeQL class
      - Description and selected predicates
    * - ``Workflow``
-     -  An Actions workflow, defined as a mapping at the top level of a workflow YAML file in `.github/workflows`. See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions.
-        - `getAJob()` - Gets a job within the `jobs` mapping of this workflow.
-        - `getEnv()` - Gets an `env` mapping within this workflow declaring workflow-level environment variables, if any.
-        - `getJob(string jobId)` - Gets a job within the `jobs` mapping of this workflow with the given job ID.
-        - `getOn()` - Gets the `on` mapping defining the events that trigger this workflow.
-        - `getPermissions()` - Gets a `permissions` mapping within this workflow declaring workflow-level token permissions, if any.
-        - `getStrategy()` - Gets a `strategy` mapping for the jobs in this workflow, if any.
-        - `getName()` - Gets the name of this workflow, if defined within the workflow.
-
-The following example lists all jobs in a workflow with the name declaration `name: test`:
+     -  An Actions workflow, defined as a mapping at the top level of a workflow YAML file in ``.github/workflows``. See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions.
```

Comment:
We have this link in line 111 above too. Suggest either deleting it here, or borrowing the formatting:

```suggestion
     -  An Actions workflow, defined as a mapping at the top level of a workflow YAML file in ``.github/workflows``. See `workflows <https://docs.github.com/en/actions/writing-workflows/about-workflows>`__ and `workflow syntax <https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions>`__.
```

Reply:
I'll come back to this. Right now the text here is richer than the text in the inline QLDoc, and they are meant to match.